app/src/main/java/com/launcher/silverfish/launcher/appdrawer/TabbedAppDrawerFragment.java

```diff
@@ -414,12 +426,19 @@ private void dropAppInTab(String app_name) {
 
     //region Extra options per app
 
-    private void showExtraOptionsMenu(final String appName) {
+    /** App drawer 'pop-up' menu after minimal drag */
+    private void showExtraOptionsMenu(final String appName, String appLabel) {
         AlertDialog.Builder builder = new AlertDialog.Builder(getContext());
 
+        // Set up a title e.g. "Silverfish (com.launcher.silverfish)"
+        String title = format("<font color='red'><b>%s</b></font> ", appLabel);
```

Comment:
Not a fan of hardcoding colours like this. Making it bold should probably be enough and would fit with the rest.